webapp/portlet/src/main/webapp/space-members/components/SpaceInvitationDrawer.vue

```diff
@@ -161,6 +163,13 @@ export default {
     },
     defaultAvatar() {
       return `${eXo.env.portal.context}/${eXo.env.portal.rest}/v1/social/users/default-image/avatar`;
+    },
+
+    invitedUser() {
+
+      return this.invitedMembers.length === 1 ? this.invitedMembers[0].userName : null;
```

Comment:
What if we invite multiple users ?

Reply:
in the task ,it wasn't clear as  they requested a message for one user , but after i discussed the problem with meriem , she told me to add the message ;" users has been invited to space" in case there is many users , so i will do it .